portfolio/src/main/webapp/background.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>About Me - Carolyn's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+     <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="background.html">About Me</a></li>
+          <li><a href="funthings.html">Some Things I like to do!</a></li>
+          <li><a href="professional.html">My Professional Page</a></li>
+          <li><a href="world.html">Travel the World With Me!</a></li>
+          <p id="date" style="float: right; padding-right: 3%"></p>
+          <script type="text/javascript"> showDate(); </script>
```

Comment:
Instead of calling this manually, we can add an event handler to the script.js to handle to `onload` event, which is fired when the browser finishes loading the page. It'll be a little cleaner and separate our html from our js.

https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event